packages/react-ui-ag/src/Listings/MobileMapListing.js

```diff
@@ -142,6 +154,30 @@ export default class MobileMapListing extends Component {
     )
   }
 
+  renderPhotoCarousel() {
+    const { theme, lazyLoad, photos } = this.props
+    const photoProps = {
+      showNav: true,
+      ...photos,
```

Comment:
Did you mean to spread these here?

Reply:
Good catch - I only meant to spread it below.

Reply:
@tadjohnston so apparently I should have keep the spread - it was just recently changed in the ListingComponents Photo component (and getting spread there as well in the destructuring assignment).

Reply:
I'm confused. If it's spread on the `ListingComponents.Photo`, why would you spread it here too?

Reply:
So I think we need to refactor how we are using and spreading props from ListingCarousel down to the ListingComponents. There are duplicate prop names and levels of sharing that make it confusing to know exactly what's getting passed around. It'll be easier if I show you.